docs/domain/03_aggregates.md

```diff
@@ -0,0 +1,63 @@
+# Aggregates
+
+Aggregates are part of DDD. To create one you have to create entity class inheriting [`IAggregateRoot`](../../src/Domain/LeanCode.DomainModels/Model/IAggregateRoot.cs) or [`IAggregateRootWithoutOptimisticConcurrency`](../../src/Domain/LeanCode.DomainModels/Model/IAggregateRoot.cs).
+
+## [`IAggregateRootWithoutOptimisticConcurrency`](../../src/Domain/LeanCode.DomainModels/Model/IAggregateRoot.cs)
```

Comment:
Let's start with `IAggregateRoot`. Then, describe the `W/oOC` version separately as a diff to `IAggregateRoot`, not the other way around.

Optimistic Concurrency should be the default here.